src/models/status/StatusReport.js

```diff
@@ -7,7 +7,7 @@ export default class StatusReport {
 
   log(logger) {
     logger.error(this.description)
-    logger.error(` - expected: ${this.expected}`)
-    logger.warn(` - actual:   ${this.observed}\n`)
+    logger.error(` - on-chain: ${this.expected}`)
+    logger.warn(` - local:   ${this.observed}\n`)
```

Comment:
This not totally right due to a bad naming of the `StatusReport` variables. Note that we are currently using `observed` to refer to what was found on-chain and `expected` to what was found locally. Then we have two alternatives:

1- Swapping the declaration of these variables in the constructor, it should be: 

```js
export default class StatusReport {
  constructor(observed, expected, description) {
    this.observed = observed
    this.expected = expected
    this.description = description
  }
...
}
```

But you will have to fix many tests. I understand if you don't want to do this.

2- We can fix your change to use `observed` to output whatever was found **on-chain** and `expected` to show what was found **locally**. And we can fix the naming issue later.